Makefile

```diff
@@ -1,209 +1,18 @@
-# Old-skool build tools.
-#
-# Targets (see each target for more information):
-#   all: Build code.
-#   build: Build code.
-#   check: Run verify, build, unit tests and cmd tests.
-#   test: Run all tests.
-#   run: Run all-in-one server
-#   clean: Clean up.
+all: build
+.PHONY: all
 
-OUT_DIR = _output
-OS_OUTPUT_GOPATH ?= 1
+build:
+	go build -o _output/bin/dockerregistry github.com/openshift/image-registry/cmd/dockerregistry
+.PHONY: build
 
-export GOFLAGS
-export TESTFLAGS
-# If set to 1, create an isolated GOPATH inside _output using symlinks to avoid
-# other packages being accidentally included. Defaults to on.
-export OS_OUTPUT_GOPATH
-# May be used to set additional arguments passed to the image build commands for
-# mounting secrets specific to a build environment.
-export OS_BUILD_IMAGE_ARGS
+build-image: build
+	hack/build-image.sh
+.PHONY: build-image
 
-# Tests run using `make` are most often run by the CI system, so we are OK to
-# assume the user wants jUnit output and will turn it off if they don't.
-JUNIT_REPORT ?= true
-
-# Build code.
-#
-# Args:
-#   WHAT: Directory names to build.  If any of these directories has a 'main'
-#     package, the build will produce executable files under $(OUT_DIR)/local/bin.
-#     If not specified, "everything" will be built.
-#   GOFLAGS: Extra flags to pass to 'go' when building.
-#   TESTFLAGS: Extra flags that should only be passed to hack/test-go.sh
-#
-# Example:
-#   make
-#   make all
-#   make all WHAT=cmd/oc GOFLAGS=-v
-all build:
-	hack/build-go.sh cmd/dockerregistry $(GOFLAGS)
-.PHONY: all build
-
-# Build the test binaries.
-#
-# Example:
-#   make build-tests
-build-tests: 
-.PHONY: build-tests
-
-# Run core verification and all self contained tests.
-#
-# Example:
-#   make check
-check: | build verify
-	$(MAKE) test-unit -o build -o verify
-.PHONY: check
-
-
-# Verify code conventions are properly setup.
-#
-# TODO add verifying listers - we can't do it yet because there's an issue with the generated
-# expansion file being incorrect.
-#
-# Example:
-#   make verify
 verify: build
-	# build-tests task has been disabled until we can determine why memory usage is so high
-	{ \
-	hack/verify-gofmt.sh ||r=1;\
-	hack/verify-govet.sh ||r=1;\
-	exit $$r ;\
-	}
+	go test github.com/openshift/image-registry/pkg/...
```

Comment:
not entirely sure the team working on this wants to give up jUnit output

Reply:
we do not, but i think this is basically a point in time "get something working" and then build back up to full function.

consider it an opportunity to start fresh w/o some of the baggage we've accrued in origin.